Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js

```diff
@@ -895,6 +896,79 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
         },
     });
 
+    navigateToFirstItemTest = new Y.Test.Case({
+        name: "eZ Navigation Hub View Service navigate to first item tests",
+
+        setUp: function () {
+            var that = this;
+
+            this.zoneName = "midfield";
+
+            this.routeName = "Lass";
+            this.routeParams = [];
+            this.item = new Y.Base();
+
+            this.item.set('route', {
+                name: this.routeName,
+                params: this.routeParams
+            });
+
+            this.parameterItem = {
+                config: {
+                    route: {
+                        name: this.routeName,
+                        params: this.routeParams,
+                    }
+                }
+            };
+
+            this.appMock = new Mock();
+
+            Mock.expect(this.appMock, {
+                method: 'navigateTo',
+                args: [Mock.Value.String, Mock.Value.Any],
```

Comment:
You can simplify it a bit with something like:
`args: [that.routeName, that.routeParams]`
so no need of defining `run` then